skimage/exposure/exposure.py

```diff
@@ -61,8 +61,8 @@ def _bincount_histogram(image, source_range):
     if source_range not in ['image', 'dtype']:
         raise ValueError('Incorrect value for `source_range` argument: {}'.format(source_range))
     if source_range == 'image':
-        image_min = np.min(image).astype(np.int64)
-        image_max = np.max(image).astype(np.int64)
+        image_min = int(np.min(image).astype(np.int64))
+        image_max = int(np.max(image).astype(np.int64))
```

Comment:
Are `np.min(image)` and `np.max(image)` returning Dask Arrays when `image` is a Dask Array? If so, casting to `int` will `compute`.

Reply:
Yes, that's my intention, sorry that wasn't clearer. 

The problem is: we can't use dask arrays as the values of keyword arguments without causing an error, even when the function itself can take dask arrays as the main input (we see this with `np.bincount()` a few lines later). That's something that ultimately would be very worthwhile to fix, this is a hack that works around it.

I wouldn't say it's *ideal* to have two dask computations triggered when you use the skimage histogram function, but I do think it's a heck of a lot better than not being able to use it at all. 

Finally, there is also another option that won't force computation. The scikit-image histogram function allows `source_range='dtype'` - which will guess an appropriate min/max based on the array type, no computation necessary.